src/GlobalVars.res

```diff
@@ -12,5 +12,5 @@
 @val external maxLogsPushedPerEventName: int = "maxLogsPushedPerEventName"
 let targetOrigin: string = "*"
 let isInteg = sdkUrl === "https://dev.hyperswitch.io"
-let isSandbox = sdkUrl === "https://beta.hyperswitch.io"
+let isSandbox = sdkUrl === "https://beta.hyperswitch.io" || sdkUrl === "http://localhost:9050"
```

Comment:
why this?

Reply:
so that we can pass the checks for wild cards on local also